### PR TITLE
Feat(Task-12): As a User, I want to contact the administrator of the page.

### PR DESCRIPTION
**Pull request by SamuFigueirido**

### CODE REVIEW

**Review by eloydotco — APPROVED** [review]

File: src/app/main-page/main-page.controller.js [hunk 1]
```diff
@@ -29,10 +31,27 @@ export default class MainPageController {
             .then(response => {
                 this.days = this.weatherContainerService.getListToShow(response);
             })
-        .catch(response => {
-            console.log('There is no weather for this city');
-        });
+            .catch(response => {
+                console.log('There is no weather for this city');
```
Comment (gbtec-ext-jorgevillar) [inline src/app/main-page/main-page.controller.js]: Use `console.error` in catchs. And maybe you could should print the error

File: src/app/components/contact-button/contact-dialog.component.scss [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+    md-input-container {
+        label {
+            color: white;
+        }
+
+        // border-width: 2px;
+        // border-style: solid;
+        // border-color:skyblue;
+        // border-radius: 15px;
+        // background-color: rgba(49, 49, 49, 0.603);
```
Comment (gbtec-ext-jorgevillar) [inline src/app/components/contact-button/contact-dialog.component.scss]: Remove commented code

File: src/app/components/contact-button/contact-dialog.component.scss [hunk 3]
```diff
@@ -0,0 +1,106 @@
+@import "./src/resources/styles/_colors";
+
+md-dialog {
+    height: 70%;
+    width: 49%;
+    border-width: 2px;
+    border-color: skyblue;
+    background-color: rgba(80, 80, 80, 0.753);
+    border-width: 2px;
+    border-style: solid;
```
Comment (gbtec-ext-jorgevillar) [inline src/app/components/contact-button/contact-dialog.component.scss]: the border width and color are repeated, and you can do it like this: `boder: 2px solid skyblue`

File: src/app/components/contact-button/contact-dialog.component.scss [hunk 4]
```diff
@@ -0,0 +1,106 @@
...
+    background-color: transparent;
+}
+
+md-toolbar:not(.md-menu-toolbar) {
+    padding: 20px;
+    text-shadow: -1px 0 black, 0 1px black, 1px 0 black, 0 -1px black;
+    background-color: rgba(49, 49, 49, 0.603);
+    border-width: 2px;
+    border-style: solid;
+    border-color: skyblue;
```
Comment (gbtec-ext-jorgevillar) [inline src/app/components/contact-button/contact-dialog.component.scss]: same as above, you can do it in one line

File: src/app/components/contact-button/contact-dialog.component.scss [hunk 5]
```diff
@@ -0,0 +1,106 @@
...
+
+md-input-container>md-icon {
+    color: white;
+}
+
+.btn-submit, .btn-cancel {
+    &.md-button {
+        border-style: solid;
+        border-color: skyblue;
+        border-width: 2px;
```
Comment (gbtec-ext-jorgevillar) [inline src/app/components/contact-button/contact-dialog.component.scss]: same as above

**Review by gbtec-ext-jorgevillar — CHANGES_REQUESTED** [review]

**Review by gbtec-ext-jorgevillar — APPROVED** [review]

**Pull request merged**